.github/workflows/dependent-issues.yml

```diff
@@ -20,6 +20,10 @@ on:
 
 jobs:
   check:
+    permissions:
+      issues: write
+      pull-requests: write
+      statuses: write
```

Comment:
I believe these are all unnecessary too? should be fine with just the default permissions we have

Reply:
[https://github.com/cryostatio/cryostat-web/actions/runs/6952595800/job/18916412442?pr=1153](https://github.com/cryostatio/cryostat-web/actions/runs/6952595800/job/18916412442?pr=1153)
I thought this was a permission issue?